meta-sokol-flex-common/classes/sdk_codebench_metadata.bbclass

```diff
@@ -0,0 +1,247 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: MIT
+# ---------------------------------------------------------------------------------------------------------------------
+
+# Write additiional metadata for CodeBench to the SDK when codebench-metadata is
+# in SDKIMAGE_FEATURES.
+
+inherit sdk_extra_vars codebench-environment-setup-d-hack
+
+OVERRIDES =. "${@bb.utils.contains('SDKIMAGE_FEATURES', 'codebench-metadata', 'sdk-codebench-metadata:', '', d)}"
+
+SDK_POSTPROCESS_COMMAND:prepend:sdk-codebench-metadata = "adjust_sdk_script_codebench; write_codebench_metadata;"
+
+TOOLCHAIN_HOST_TASK:append:sdk-codebench-metadata = " ${@bb.utils.contains('BBFILE_COLLECTIONS', 'sokol-flex-support', 'nativesdk-relocate-makefile', '', d)}"
+TOOLCHAIN_TARGET_TASK:append:sdk-codebench-metadata = " ${@bb.utils.contains('BBFILE_COLLECTIONS', 'sokol-flex-support', 'codebench-makefile', '', d)}"
+
+SOURCERY_VERSION ?= ""
+SDK_IDENTIFIER ?= "${SDK_NAME}"
+ADE_IDENTIFIER ?= "${SDK_IDENTIFIER}"
```

Comment:
I can see Var with ADE in it

Reply:
Currently CodeBench still expects the ADE variables, so that part hasn't been migrated yet.

Reply:
@kergoth we can tell them the Variables names has been updated. They have to make adjustment on their side. we can give the a sample SDK

Reply:
Fixed, thanks.